### ISSUE TITLE
Rendering Angular MFEs inside React Shell

### ISSUE THREAD

**Issue by patel-jeel92** [module-federation/module-federation-examples#1846]
Hi. Are there examples of rendering Angular 12 micro front ends inside a React Shell using module federation? I am having trouble integrating angular MFEs inside React Shell. React MFEs are easy to integrate and all works well. There are many examples of this scenario. 

Can we have an example of multi framework MFEs with the shell being a React App and other micro front ends being Angular?

**Comment by ScriptedAlchemy** [comment]
There's a few multi-app shell examples in this repo, just have to dig around in the various examples. 

I,m not sure if theres a angular host one, but the general principal would probably be quite similar.

**Comment by patel-jeel92** [comment]
I havent found examples of React Shell importing an angular front end or how angular app should be built with webpack. I have a prototype that I am working on right now, but i am running into issues with rendering the angular mfe

**Comment by patel-jeel92** [comment]
[React + Angular.zip](https://github.com/module-federation/module-federation-examples/files/8413006/React.%2B.Angular.zip)
Here is the poc i have been working on. It has two projects, a shell that's written in React, and another micro front end written in Angular. I am trying to render the angular micro front end inside the React shell but i have been running into issues. You can try running the project on your end to see if you see the same error:
```
Uncaught TypeError: Class constructor StaticAngularComponent cannot be invoked without 'new'
    at renderWithHooks (vendors-node_modules_react-dom_index_js.bundle.js:14812:18)
    at updateFunctionComponent (vendors-node_modules_react-dom_index_js.bundle.js:17043:20)
    at mountLazyComponent (vendors-node_modules_react-dom_index_js.bundle.js:17357:17)
    at beginWork (vendors-node_modules_react-dom_index_js.bundle.js:18611:16)
    at HTMLUnknownElement.callCallback (vendors-node_modules_react-dom_index_js.bundle.js:197:14)
    at Object.invokeGuardedCallbackDev (vendors-node_modules_react-dom_index_js.bundle.js:246:16)
    at invokeGuardedCallback (vendors-node_modules_react-dom_index_js.bundle.js:301:31)
    at beginWork$1 (vendors-node_modules_react-dom_index_js.bundle.js:23212:7)
    at performUnitOfWork (vendors-node_modules_react-dom_index_js.bundle.js:22163:12)
    at workLoopSync (vendors-node_modules_react-dom_index_js.bundle.js:22139:22)
```
# Steps to reproduce 
1) Unzip folder
2) npm install in both projects
3) npm run start for both projects 
4) Navigate to shell running on localhost:8080 and open on dev console

**Comment by ebaldebo** [comment]
We referenced this implementation when we made our poc to make Angualar work. We have a React shell with dynamically loaded remotes in React, Angular and Solidjs. The only issue we're having right now is loading assets(images/fonts etc) from the Angular application in the shell.

[https://medium.com/@harshverma04111989/webpack-5-module-federation-for-micro-frontend-architecture-ed3b77c7d729](url)
[https://github.com/harshmons/micro-front-end-react-application](url)
